pkg/reconciler/build/build.go

```diff
@@ -0,0 +1,99 @@
+// Copyright The Shipwright Contributors
```

Comment:
Any chance we could save the history of this code? I think it was moved from another file, right?

Reply:
@HeavyWombat I think I did that, but it went to `pkg/reconciler/build/controller.go`. `I´m not sure if is possible to do a `git mv` to multiple files. Opinions?